### PR TITLE
GSF WSF diagnostics

### PR DESCRIPTION
**Pull request by totto82**
Depends on https://github.com/OPM/opm-common/pull/3406

### CODE REVIEW

File: opm/core/props/satfunc/RelpermDiagnostics.cpp [hunk 1]
```diff
@@ -144,14 +149,30 @@ namespace Opm {
...
         }
 
+        bool family3 = !gsfTables.empty() && !wsfTables.empty();
+
+        if (family3) {
+            const auto& phases = eclState.runspec().phases();
+            const bool co2store = eclState.runspec().co2Storage();
+            if ( !(co2store && phases.active(Phase::GAS) && phases.active(Phase::WATER))) {
+                const std::string msg = "Relative permeability input format: Saturation Family III. \n \
+                                         Only valied for CO2STORE case with GAS and WATER.";
```
Comment (akva2) [inline opm/core/props/satfunc/RelpermDiagnostics.cpp:159]: valid

Comment (totto82) [reply opm/core/props/satfunc/RelpermDiagnostics.cpp]: Thanks!

**Comment by totto82** [comment]
jenkins build this opm-common=3406 please

**Review by bska — APPROVED** [review]
Looks good to me and I'll merge into master.

**Pull request merged**